src/hooks/useDailyPriceStats.ts

```diff
@@ -22,6 +22,37 @@ export const TOKEN_LIST = [
   Token.Bonk,
 ];
 
+export function getTokenGivenAddress(address: string) {
```

Comment:
a better name for this might be `tokenAddressToToken`. Also, might want to consider a default case, otherwise we'll be dealing with `undefined`s being returned from this method. Or, if that is ok, still having an explicit default case and returning `null` would be preferable